site/docs/iceberg/trino.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Nessie + Iceberg + Trino"
+---
+
+# Trino via Iceberg
+
+Trino comes with Iceberg and Nessie catalog clients installed by default. 
+You can deploy Trino using any of the methods mentioned in [this document](https://trino.io/docs/current/installation.html) and to become familiar with the Iceberg connector, refer [this document](https://trino.io/docs/current/connector/iceberg.html?highlight=iceberg+connector).
+Nessie catalog properties required for the Iceberg connector can be found at https://trino.io/docs/current/object-storage/metastores.html#nessie-catalog.
+
+Sample Nessie configuration for Iceberg connector in `etc/catalog/iceberg.properties`
+```
+connector.name=iceberg
+iceberg.catalog.type=nessie
+iceberg.nessie-catalog.uri=http://localhost:19120/api/v2
+iceberg.nessie-catalog.default-warehouse-dir=/Users/foo/wh
```

Comment:
is this a hive/hadoop warehouse dir for local development?
wondering if an S3 based example would help more people (i.e. does it mean also need to configure fileio? i am not familiar with trino)

Reply:
Trino doesn't use Iceberg FileIO, we need to configure the credentials as mentioned in this section
https://trino.io/docs/current/connector/iceberg.html?highlight=s3+aws+access+key#file-system-access-configuration

Reply:
updated with s3 warehouse path.